core/templates/components/forms/schema-based-editors/schema-based-editor.directive.ts

```diff
@@ -99,7 +99,7 @@ export const ScopeProvider = {
 export class SchemaBasedEditorDirective extends UpgradeComponent {
   @Input() schema: () => Schema;
   @Input() isDisabled: () => boolean;
-  @Input() localValue;
+  @Input() localValue: unknown;
```

Comment:
would it be better to have a union type of the possible values here? There should be only a small set of valid values that you can assign to this right?

Reply:
@nithusha21 I only thought of it as `string` ,couldn't find any instances where some other type is passed .Also most parts of code using this property is in ajs ,so it is difficult to determine type.Later @srijanreddy98 suggested to make it unknown.

Reply:
I only thought of it as a string, couldn't find any instances where some other type was passed. @srijanreddy98 suggested to make it unknown.

Reply:
Not covering this file so removed it for now.